### DISCUSSION TITLE
mass upsert

### DISCUSSION THREAD

**Discussion by Adeelasd** [prisma/prisma#6451] [Q&A]
is it possible to mass upsert data?

**Comment by ryands17** [comment]
@Adeelasd 👋 

Unfortunately not, but it would be great if you add a 👍 to this request so that we can check the priority for this :)

https://github.com/prisma/prisma/issues/4134

As a workaround, you would need to use `prisma.$transaction` with `upsert` for multiple upserts.

**Reply by Adeelasd** [reply]
yes that is what i ended up doing.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
